manifest.json

```diff
@@ -1,27 +1,32 @@
 {
     "name": "__MSG_extensionName__",
     "author": "Talis Education Limited",
-    "description":"__MSG_description__",
+    "description": "__MSG_description__",
     "version": "__VERSION__",
-    "icons":  {
+    "icons": {
         "16": "images/icon16.png",
         "48": "images/icon48.png",
         "128": "images/icon128.png",
         "176": "images/icon176.png"
     },
     "background": {
-        "page": "background.html",
-        "persistent": false
+        "service_worker": "background.js",
+        "type": "module"
     },
     "permissions": [
-        "activeTab", "storage", "https://talis-public.talis.com/*"
+        "activeTab",
```

Comment:
Needed to add a couple more permissions for the new apis we must use, `scripting` and `tabs`